src/packages.js

```diff
@@ -53,7 +58,7 @@ import {createPackageAvailabilityCheck} from './utils/packages';
  * @property {boolean} [hidden] Hide from launch menus etc.
  * @property {string} [server] Server script filename
  * @property {string[]} [groups] Only available for users in this group
- * @property {string[]} [files] Files to preload
+ * @property {Object[]|string[]} [files] Files to preload
```

Comment:
I think the only thing left here is to update the TS type definition :blush:

Reply:
I actually have a script to generate this from the esdoc/jsdoc... I need to add that stuff into the repo.